### ISSUE TITLE
config config_new_model_experiment.tsv is missing many required params

### ISSUE THREAD

**Issue by taylorreiter** [greenelab/generic-expression-patterns#114]
For the command `train_vae_modules.train_vae(config_filename, normalized_compendium_filename)`, the following params were missing (i stole values from another config file in the repo):

```
learning_rate   0.001
batch_size      10
kappa   .01
intermediate_dim        2500
epsilon_std     1.0
epochs  100
validation_frac .25
```

For the code block 
```
# Create VAE directories if needed
output_dirs = [
    os.path.join(base_dir, dataset_name, "models"),
    os.path.join(base_dir, dataset_name, "logs"),
]

NN_architecture = params["NN_architecture"]

# Check if NN architecture directory exist otherwise create
for each_dir in output_dirs:
    sub_dir = os.path.join(each_dir, NN_architecture)
    os.makedirs(sub_dir, exist_ok=True)
```
the following params were missing:
```
NN_architecture "NN_2500_30"
```

For other places in the nb, the following were missing/needed to be changed:
```
processed_compendium_filename
sample_id_colname = params["sample_id_colname"]
```

And I think this was asked for, but was irrelevant:
```
shared_genes_filename = params["shared_genes_filename"]
```

**Comment by ajlee21** [comment]
Addressed in https://github.com/greenelab/sophie/pull/3
